Masarin.IoT.Sensor/Decoders/MQTTDecoderLoRaWAN.cs

```diff
@@ -122,20 +122,36 @@ public override void Decode(string timestamp, string device, string topic, byte[
             {
                 if (obj.ContainsKey("co2"))
                 {
-                    double value = obj.co2;
-                    var strValue = $"co2%3D{value}";
-                    var msg = new Fiware.DeviceMessage(deviceName, strValue);
+                    double co2 = obj.co2;
+                    DateTime dateObserved = DateTime.UtcNow;
+                    var aqoMsg = new Fiware.AirQualityObserved(deviceName, dateObserved.ToString());
+                    aqoMsg = aqoMsg.WithCO2(co2);
+
+                    if (obj.ContainsKey("temperature")) 
+                    {
+                        double temp = obj.temperature;
+                        aqoMsg = aqoMsg.WithTemperature(temp);
+                    }
+
+                    if (obj.ContainsKey("humidity")) 
+                    {
+                        double humidity = obj.humidity;
+                        aqoMsg = aqoMsg.WithHumidity(humidity);
```

Comment:
Relaative humidity is encoded as a value between 0 and 1 so you need to divide with 100.0 here.